WalletWasabi.Gui/Controls/WalletExplorer/CoinViewModel.cs

```diff
@@ -113,6 +116,21 @@ public CoinViewModel(CoinListViewModel owner, SmartCoin model)
 				.DisposeWith(Disposables);
 
 			ToggleDetails = ReactiveCommand.Create(() => IsExpanded = !IsExpanded);
+			
+			OpenCoinInfo = ReactiveCommand.Create(() =>
+			{ 
+				var shell = IoC.Get<IShell>();
+
+				var coinInfo = shell.Documents?.OfType<CoinInfoTabViewModel>()?.FirstOrDefault(x => x.Coin?.TransactionId == TransactionId);
```

Comment:
Seems like I should use OutputIndex instead of `TransactionId` for determining output coin uniqueness right? @lontivero @nopara73 @molnard

Reply:
On business logic level I would use the SmartCoin itself as an identifier.

Reply:
The coin uniqueness is given by what is called `outpoint`, this is `txid + index`. Remember you can have many coins from the same transaction so, transaction id is not enough.